video/out/vo_vaapi_wayland.c

```diff
@@ -266,10 +243,6 @@ static int preinit(struct vo *vo)
         p->display = NULL;
         goto fail;
     }
-    va_get_formats(p);
-    if (!p->image_formats)
-        goto fail;
```

Comment:
Are you sure this isn't needed? The other `vo_vaapi` driver has this code.

Reply:
Thanks for the review - I've added more info in the commit and PR description for why this is not needed.